mathesar/models/base.py

```diff
@@ -57,6 +57,7 @@
 
 
 NAME_CACHE_INTERVAL = 60 * 5
+CACHED_PREVIEW_FOR_TABLES = {}
```

Comment:
Rename to something that expresses that you're not caching the actual preview, but just the id of the column from which preview will be built.